data/remnant/remnant missions.txt

```diff
@@ -1062,6 +1062,141 @@ mission "Remnant: Learn Sign Follow Up"
+			label investigatechoice
+			`	Thinking carefully, he says, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`
+			choice
+				`	"I'd be happy to."`
+				`	"I'd prefer not to."`
+					decline
+			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
+				accept
+	on complete
+		conversation
+			`Arriving on Bounty, you take a discreet look around the spaceport, but you don't find the researchers or any sign of them. It appears that they packed up and left since the last time you saw them. Time to look around for some info.`
+
+
+
+mission "Remnant: Follow-up on Terminus Researchers A-2"
+	name "Check up on Terminus Researchers"
+	description "Tracking down the wormhole researchers."
+	source "Bounty"
+	destination
+		government "Deep"
+	to offer
+		has "Remnant: Follow-up on Terminus Researchers A-1: done"
+	on offer
+		conversation
+			`A few hours of searching later, you find a lead: It seems that launching probes into the Terminus system isn't exactly a common activity, so their permits stand out from the rest. A helpful clerk in the spaceport office pulled up their application and found that they are listed as part of a university in the Deep, although it doesn't mention which planet their specific campus is on. A quick check of the directory lists a number of planets where they could be found.`
+			choice
+				`	(Stop investigating.)`
+					decline
+				`	(Continue the investigation)`
+					accept
+	on complete
+		conversation
+			`Once you've landed on <planet>, you look up the local university and pay it a visit. After thinking about it for a minute, you decide that the best source of information would probably be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief searching, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`
+
+
+
+mission "Remnant: Follow-up on Terminus Researchers AB-3"
+	name "Check up on Terminus Researchers"
+	description "Returning with information on the wormhole researchers."
+	source
+		government "Deep"
+	destination
+		government "Remnant"
+	to offer
+		or
+			has "Remnant: Follow-up on Terminus Researchers A-2: done"
+			has "Remnant: Follow-up on Terminus Researchers B: done"
+	on offer
+		conversation
+			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the abstract is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`
```

Comment:
Do they still use physical paper?  The previous part makes it sound like a digital transfer (can't really flip through digital media).

Reply:
It could be actual papers; but that being said, I flip through pdf's all the time, even if there is no actual paper.

Could be something as simple as being an old term that has stuck with humanity, much like present day operating systems frequently use a little disk icon to represent "save", despite the fact that we have large swaths of computer-using people who have never even seen a floppy disk, let alone used one regularly as a means of saving work.